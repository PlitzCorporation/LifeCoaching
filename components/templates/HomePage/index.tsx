import React from 'react';

import PlitzLogo from '@/components/logos/PlitzLogo';

const HomePage = () => {
	return (
		<main className="flex min-h-screen flex-col items-center justify-between p-10 lg:p-24">
			<h1 className="text-center text-balance w-2/3 mb-7">Reason 4 Hope</h1>
			<div className="w-full lg:w-2/4 flex flex-col items-center justify-center">
				<h2 className="text-center mb-7">Website development in progress</h2>
			</div>
			<div className="flex flex-col justify-center items-center text-center">
				<PlitzLogo className="w-14" />
				<p className="py-4">Plitz Corporation Â© 2024</p>
			</div>
		</main>
	);
};

export default HomePage;
