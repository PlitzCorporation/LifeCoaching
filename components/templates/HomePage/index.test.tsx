import React from 'react';
import { render } from '@testing-library/react';
import HomePage from './index';

describe('HomePage', () => {
	it('renders the welcome message', () => {
		const { container } = render(<HomePage />);
		expect(container.firstChild).toMatchSnapshot();
	});
});
