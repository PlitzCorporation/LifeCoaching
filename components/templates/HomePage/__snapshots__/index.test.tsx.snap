// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage renders the welcome message 1`] = `
<main
  class="flex min-h-screen flex-col items-center justify-between p-10 lg:p-24"
>
  <h1
    class="text-center text-balance w-2/3 mb-7"
  >
    Reason 4 Hope
  </h1>
  <div
    class="w-full lg:w-2/4 flex flex-col items-center justify-center"
  >
    <h2
      class="text-center mb-7"
    >
      Website development in progress
    </h2>
  </div>
  <div
    class="flex flex-col justify-center items-center text-center"
  >
    <svg
      class="w-14"
      viewBox="0 0 110 25"
      xmlns="http://www.w3.org/2000/svg"
    >
      <defs>
        <clippath
          id="a"
        >
          <path
            d="M0 0h108.363v25H0z"
            data-name="Rectangle 4"
            fill="#efefef"
          />
        </clippath>
      </defs>
      <g
        clip-path="url(#a)"
        fill="#efefef"
      >
        <path
          d="M11.548 17.791H4.619V25H0V.504c3.849 0 7.7-.035 11.548-.035 11.968 0 12 17.322 0 17.322m-6.929-4.236h6.929c5.914 0 5.879-8.679 0-8.679H4.619Z"
          data-name="Path 1"
        />
        <path
          d="M26.297.504v20.227h12.6V25H21.678V.504Z"
          data-name="Path 2"
        />
        <path
          d="M40.206.504h4.584V25h-4.584z"
          data-name="Rectangle 2"
        />
        <path
          d="M53.816 4.703h-7.769v-4.2h20.157v4.2h-7.769v20.3h-4.619Z"
          data-name="Path 3"
        />
        <path
          d="M67.037.504h18.827v2.17l-12.7 17.952h12.7V25H66.722v-2.66l12.423-17.5H67.037Z"
          data-name="Path 4"
        />
        <path
          d="M89.536 0h18.827v2.17l-12.7 17.952-5.2-.024L101.644 4.34H89.536Z"
          data-name="Path 5"
        />
        <path
          d="M86.456 6.854h2.766v10.787h-2.766z"
          data-name="Rectangle 3"
        />
      </g>
    </svg>
    <p
      class="py-4"
    >
      Plitz Corporation Â© 2024
    </p>
  </div>
</main>
`;
